jwst/stpipe/tests/steps/__init__.py

```diff
@@ -34,6 +34,35 @@ def process(self, a=0, b=0):
         return a + b
 
 
+class MakeListStep(Step):
+    """Make a list of all arguments and parameters"""
+
+    spec = """
+    par1 = float() # Control the frobulization
```

Comment:
Does the frobulization control the illudium q-36 explosive space modulator?

Reply:
![67d36be11f5a769e0c8b367c834763fa](https://user-images.githubusercontent.com/17088869/61466174-71e33a80-a947-11e9-963d-e99491d1d595.gif)

Reply:
*meep* *meep*